### ISSUE TITLE
Symbol's value as variable is void: package-recipe--working-tree

### ISSUE THREAD

**Issue by DamienCassou** [cask/cask#420]
When installing Cask with

```
curl -fsSkL https://raw.github.com/cask/cask/master/go | python
```

I now get 

```
In toplevel form:
package-recipe.el:67:1:Warning: `(rcp package-recipe)' is a malformed function
package-recipe.el:67:15:Warning: reference to free variable
    `package-recipe--working-tree'
package-recipe.el:69:28:Warning: reference to free variable `rcp'
package-recipe.el:71:1:Warning: `(rcp package-recipe)' is a malformed function
package-recipe.el:71:15:Warning: reference to free variable
    `package-recipe--upstream-url'
In end of data:
package-recipe.el:109:1:Warning: the function `cl-defmethod' is not known to
    be defined.
Wrote /home/travis/.emacs.d/.cask/24.3/bootstrap/package-build-20180324.2042/package-recipe.elc
Done (Total of 1 file compiled, 3 failed, 2 skipped)
Symbol's value as variable is void: package-recipe--working-tree
Cask could not be bootstrapped. Try again later, or report an issue at https://github.com/cask/cask/issues
```

**Comment by Wilfred** [comment]
I've hit this too: https://travis-ci.org/Groovy-Emacs-Modes/groovy-emacs-modes/jobs/358324208

**Comment by anildigital** [comment]
```
root@ubuntu-s-1vcpu-1gb-blr1-01:~# curl -fsSL https://raw.githubusercontent.com/cask/cask/master/go | python
Cloning into '/root/.cask'...
remote: Counting objects: 3518, done.
remote: Total 3518 (delta 0), reused 0 (delta 0), pack-reused 3518
Receiving objects: 100% (3518/3518), 1.12 MiB | 338.00 KiB/s, done.
Resolving deltas: 100% (1980/1980), done.
Checking connectivity... done.
Importing package-keyring.gpg...
Importing package-keyring.gpg...done
Contacting host: melpa.org:443
Contacting host: melpa.org:443
Contacting host: elpa.gnu.org:443
Contacting host: elpa.gnu.org:443
Contacting host: elpa.gnu.org:443
Symbol's value as variable is void: package-recipe--working-tree
Cask could not be bootstrapped. Try again later, or report an issue at https://github.com/cask/cask/issues
root@ubuntu-s-1vcpu-1gb-blr1-01:~# lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.4 LTS
Release:	16.04
Codename:	xenial
```
Same error here.

**Comment by anildigital** [comment]
Any update on this?

**Comment by anildigital** [comment]
@Wilfred How did you fix this?

**Comment by Wilfred** [comment]
I haven't found a fix.

**Comment by Wilfred** [comment]
For what it's worth, I've restarted a Travis build and it's now green: https://travis-ci.org/Groovy-Emacs-Modes/groovy-emacs-modes/builds/358327128

**Comment by anildigital** [comment]
@Wilfred I can confirm it’s not fixed yet. I am trying to install Cask on fresh Ubuntu machine.

**Comment by sambrightman** [comment]
On the surface this sounds like it will be related to #423.

**Comment by Fuco1** [comment]
This has been fixed in #423.
